tests/utils_tests/test_lorem_ipsum.py

```diff
@@ -1,16 +1,106 @@
 import unittest
+from unittest.mock import patch
 
-from django.utils.lorem_ipsum import paragraphs, words
+from django.utils.lorem_ipsum import paragraph, paragraphs, sentence, words
 
 
 class WebdesignTest(unittest.TestCase):
 
-    def test_words(self):
+    def test_negative_words(self):
+        """Common words when passing negative words"""
+        self.assertEqual(words(-5), 'lorem ipsum dolor sit amet consectetur adipisicing elit sed do eiusmod '
+                                    'tempor incididunt ut')
+
+    def test_same_or_less_common_words(self):
+        """The first n standard lorem ipsum words"""
         self.assertEqual(words(7), 'lorem ipsum dolor sit amet consectetur adipisicing')
 
+    def test_common_words_in_string(self):
+        """Starts with all the standard lorem ipsum words"""
+        self.assertTrue(words(25).startswith('lorem ipsum dolor sit amet consectetur adipisicing elit sed do eiusmod '
+                                             'tempor incididunt ut labore et dolore magna aliqua'))
+
+    def test_more_words_than_common(self):
+        """The right amount of words is returned"""
+        self.assertEqual(len(words(25).split()), 25)
+
+    def test_common_crazy_amount_of_words(self):
+        """String has n words when n is greater than WORDS length"""
+        self.assertEqual(len(words(500).split()), 500)
+
+    @patch('django.utils.lorem_ipsum.random.sample')
+    def test_not_common_words(self, mock_sample):
+        """Words return the values expected."""
+        mock_sample.return_value = ['exercitationem', 'perferendis']
+        self.assertEqual('exercitationem perferendis', words(2, False))
```

Comment:
I guess it's better to have ` self.assertEqual(words(2, False), 'exercitationem perferendis')` to follow the pattern of `(value, expected)`